index.html

```diff
@@ -1,17 +1,137 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Page Title</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1'>
-    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
-    <script src='main.js'></script>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Navigation bar</title>
 </head>
 
 <body>
-    <h1>hello</h1>
+
+    <header>
+        <div id="logo"><a href="/"><img src="images/logo.svg"></a></div>
+        <div id="topLinks">
+            <nav>
+                <ul>
+                    <li>
+                        <div class="dropdown">
+                            <button class="dropbtn">Features
+                                <i class="fa fa-caret-down"></i>
+                            </button>
+                            <div class="dropdown-content">
+                                <a href="#"><img src="/images/icon-todo.svg"><span class="dropdown-content-span"> ToDo
```

Comment:
The dropdown menus should also be wrapped in `ul` and `li` elements.
Checkout this example https://css-tricks.com/solved-with-css-dropdown-menus/